app/frontend/src/styles/application.scss

```diff
@@ -58,15 +50,9 @@ dd {
   margin: 0;
 }
 
-.govuk-heading {
-  &--summary-list {
-    border-bottom: 6px solid $govuk-border-colour;
-    padding-bottom: govuk-spacing(2);
-  }
-}
-
-.govuk-footer__list {
-  padding-bottom: 40px;
+ul li + br,
+ul > br {
+  display: none;
```

Comment:
these styles are very good example of visual testing being useful, they are so generic, im not what they do, what might change if i remove them! i cant search codebase for a classname